### PR TITLE
Added additional information to event-update page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information about how to delete an event and buttons bar.
Replaced the screenshot with new more accurate. Previous screenshot contained an information from creating new event entry instead of update existing event entry

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/event-update.html

### CODE REVIEW

File: src/marketing/event-update.md [hunk 1]
```diff
@@ -29,3 +29,24 @@ _Edit Event_
 1. Make the necessary changes to the event.
 
 1. When complete, click <span class="btn">Save</span>.
+
+## Delete an event
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Private Sales_ > **Events**.
+
+1. Find the event in the list, and open it in edit mode.
```
Comment (BarnyShergold) [inline src/marketing/event-update.md:37]:
```suggestion
1. Find the event in the list and open it in edit mode.
```

File: src/marketing/event-update.md [hunk 2]
```diff
@@ -29,3 +29,24 @@ _Edit Event_
...
+1. In the upper-right corner, click <span class="btn">Delete</span>.
+
+1. To confirm the action click <span class="btn">OK</span>.
+
+### Button bar
+
+{: .buttons-table }
+|Button|Description|
+|--- |--- |
+|<span class="btn">Back</span>|Returns to the Events page without saving the new event or changes in existing.|
```
Comment (BarnyShergold) [inline src/marketing/event-update.md:48]:
```suggestion
|<span class="btn">Back</span>|Returns to the Events page without saving the new event or changes in an existing event.|
```

File: src/marketing/event-update.md [hunk 3]
```diff
@@ -29,3 +29,24 @@ _Edit Event_
...
+|<span class="btn">Delete</span>|Deletes the event.
+|<span class="btn">Reset</span>|Clears the form of any unsaved changes, and restores the original event information.|
```
Comment (BarnyShergold) [inline src/marketing/event-update.md:50]:
```suggestion
|<span class="btn">Reset</span>|Clears the form of any unsaved changes and restores the original event information.|
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few minor corrections.

File: src/marketing/event-update.md [hunk 4]
```diff
@@ -29,3 +29,24 @@ _Edit Event_
...
+|<span class="btn">Save and Continue Edit</span>|Saves all changes, and keeps the form open in edit mode.|
```
[the remark this answers is not in the document]
Comment (DmitryTsymbal) [reply src/marketing/event-update.md]: Thanks for the review! Changes applied.

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**